packages/i18n/I18nDecorator/I18nDecorator.js

```diff
@@ -81,6 +114,18 @@ const defaultConfig = {
 const I18nDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	const {latinLanguageOverrides, nonLatinLanguageOverrides, sync} = config;
 
+	// Normalize the structure of the external resources to be an array of resource/onLoad pairs
+	const extResources = Array.isArray(config.resources) ? config.resources.map(res => {
+		if (!res) return;
+
+		const fn = res.resource || res;
+		const onLoad = res.onLoad;
+
+		if (typeof fn !== 'function') return;
+
+		return {resource: fn, onLoad};
+	}).filter(Boolean) : [];
```

Comment:
Food for thought question on these cases where we `map().filter(Boolean)`.  Would these be clearer/easier with `reduce()` instead?  I don't know but it seems like they're pretty similar.